build.sh

```diff
@@ -43,12 +43,12 @@ export STARROCKS_HOME=${ROOT}
 . ${STARROCKS_HOME}/env.sh
 
 if [[ ! -f ${STARROCKS_THIRDPARTY}/installed/include/fast_float/fast_float.h ]]; then
-    echo "Thirdparty libraries need to be build ..."
+    echo "Thirdparty libraries need to be build because fast_float has not been built yet ..."
     ${STARROCKS_THIRDPARTY}/build-thirdparty.sh
 fi
 
 if [[ ! -f ${STARROCKS_THIRDPARTY}/installed/include/pulsar/Client.h ]]; then
-    echo "Thirdparty libraries need to be build ..."
+    echo "Thirdparty libraries need to be build because pulsar has not been built yet ..."
     ${STARROCKS_HOME}/thirdparty/build-thirdparty.sh
 fi
```

Comment:
I think there is no need to keep these two checks.
Keeps only one is enough to make sure that the third-party has been successfully built.
The last installed one of the third-party build processing could be better.